Site/CuratedContentEditorialWorkflow.md

```diff
@@ -56,15 +56,22 @@ each of the numbered steps in the description below.
    * Any issues with sufficient interest are moved to *In Progress*. An *In Progress* issue
      must have an **Author** and an **EB Member** who is different from the **Author**.
      The **EB Member** will be the assignee of the issue. The **Author** will be indicated
-     by a reference at the top of the first comment in the issue (*not* an @ reference). Finally, a
-     deadline is set by assigning a milestone.
+     by a reference at the top of the first comment in the issue (*not* an @ reference, 
+     use "Author: Foo1, Foo2"). Finally, a deadline is set by assigning a milestone.
```

Comment:
That does not allow you to search for "Author: Foo2".

Reply:
Hmmm...I don't know why that is. Is the problem that `Author` is a GitHub keyword? Because, on #426 I clearly show with this search, [`is:issue is:open "content_creators: Mark S."`](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=✓&q=is%3Aissue+is%3Aopen+%22content_creators%3A+Mark+S.%22 ), you *can* do a successful search on a subset of authors separated by commas.

Reply:
@markcmiller86, I have been trying to explain this but that query is not doing what you think.  Check out this query [is:issue is:open "Author: Mark" ](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+%22Author%3A+Mark%22+).  That is matching more issues than it should if it was actually matching the string "Author: Mark".  For example, it lists the matching issue #259 but the string "Author: Mark" is no were in that issue.  Instead, it seems to be matching "Author" and "Mark" as separate strings (and not even the ':'' in "Author:")!!!!

I am sure I can come up with other examples but hopefully this makes the point.

Can someone please verify?

Therefore, I think this needs to be:

```
Author:Mark
Author:Ross
```

with no spaces!

Reply:
Sheesh! I hadn't realized that my test case wasn't considering those alternative matches 😞 .

> Can someone please verify?

No need. I confirm that behavior and sorry for misleading.